hub3/models/sparql_test.go

```diff
@@ -1,93 +0,0 @@
-// Copyright © 2017 Delving B.V. <info@delving.eu>
```

Comment:
Why did this test become obsolete?

Reply:
Moved to the sparql service. This will be removed when the migration is completed.